src/Compute/v2/Models/QuotaSet.php

```diff
@@ -0,0 +1,170 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenStack\Common\Resource\Deletable;
+use OpenStack\Common\Resource\OperatorResource;
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\Updateable;
+
+/**
+ * Represents a Compute v2 Quota Set
+ *
+ * @property \OpenStack\Compute\v2\Api $api
+ */
+class QuotaSet extends OperatorResource implements Retrievable, Updateable, Deletable
+{
+    /**
+     * The number of allowed instance cores for each tenant.
+     *
+     * @var  int|array
```

Comment:
Hmm, that's a crappy UX for consumers of that API. 

How about we introduce a `Quota` model that encapsulates `in_use`, `limit` and `reserved` fields? If the API returns just an int (e.g. `os-quota-sets/{tenantId}`), that populates just the `limit` field?